landa/landa_sales/sales_order/sales_order.py

```diff
@@ -18,8 +18,20 @@ def before_validate(sales_order, event):
 	):
 		sales_order.tax_category = "Umsatzsteuer"
 
+
 def autoname(doc, event):
 	"""Create Company-specific Sales Order name."""
 	from landa.utils import get_new_name
 
 	doc.name = get_new_name("BEST", doc.company, "Sales Order")
+
+
+def get_dashboard_data(data):
+	custom_transactions = [
+		{
+			"label": "Sales",
+			"items": ["Delivery Note", "Sales Invoice"],
+		}
+	]
+	data["transactions"] = custom_transactions
```

Comment:
```suggestion
	data["transactions"] = [
		{
			"label": "Sales",
			"items": ["Delivery Note", "Sales Invoice"],
		},
	]
```

I liked this style as well, but variables that are only used once, are discouraged in best practices.

Reply:
https://docs.sourcery.ai/Reference/Built-in-Rules/refactorings/inline-variable/